### PR TITLE
Updated docs for path-params

### PR DESCRIPTION
**Pull request by yankeexe**
* Added example response for enum section. 
* Added URL's for 
 * Redoc in localhost
 * Starlett
* Added line breaks in `tutorial005` for readability.

### CODE REVIEW

**Comment by tiangolo** [comment]
Cool, thanks! :cake: :memo:

File: docs/en/docs/tutorial/path-params.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ And when you open your browser at <a href="http://127.0.0.1:8000/docs" class="ex
 
 And because the generated schema is from the <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md" class="external-link" target="_blank">OpenAPI</a> standard, there are many compatible tools.
 
-Because of this, **FastAPI** itself provides an alternative API documentation (using ReDoc):
+Because of this, **FastAPI** itself provides an alternative API documentation (using ReDoc), which you can access at <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>:
```
Comment (Kludex) [inline docs/en/docs/tutorial/path-params.md:88]: I'm not sure that we can assume the existence of this URL.

Comment (Kludex) [reply docs/en/docs/tutorial/path-params.md:88]: it was already merged, nvm

Comment (tiangolo) [reply docs/en/docs/tutorial/path-params.md:88]: Yeah, I was thinking that, but the example right above for Swagger UI assumes it's running (from the previous examples), so I guess it makes sense to have the link here too.

**Pull request merged**